### ISSUE TITLE
2.6 pre5 The bold formatting effect doesn't look good.

### ISSUE THREAD

**Issue by WengerChan** [kingToolbox/WindTerm#1991]
![image](https://github.com/kingToolbox/WindTerm/assets/37797982/f1215110-0993-45ec-baed-c0c94b31936c)

**Comment by WengerChan** [comment]
too big and not harmonious.

**Comment by kingToolbox** [comment]
Yes, I will fix this issue in the next `2.6.0 Prerelease`. Please stay tuned, thank you.
